exchanges/btse/btse_websocket.go

```diff
@@ -191,28 +167,36 @@ func (b *BTSE) GenerateDefaultSubscriptions() {
 
 // Subscribe sends a websocket message to receive data from the channel
 func (b *BTSE) Subscribe(channelToSubscribe wshandler.WebsocketChannelSubscription) error {
-	subscribe := websocketSubscribe{
-		Type: "subscribe",
-		Channels: []websocketChannel{
-			{
-				Name:       channelToSubscribe.Channel,
-				ProductIDs: []string{channelToSubscribe.Currency.String()},
-			},
-		},
-	}
-	return b.WebsocketConn.SendMessage(subscribe)
+	var sub wsSub
+	sub.Operation = "subscribe"
+	sub.Arguments = []string{channelToSubscribe.Channel}
+	return b.WebsocketConn.SendMessage(sub)
 }
 
 // Unsubscribe sends a websocket message to stop receiving data from the channel
 func (b *BTSE) Unsubscribe(channelToSubscribe wshandler.WebsocketChannelSubscription) error {
-	subscribe := websocketSubscribe{
-		Type: "unsubscribe",
-		Channels: []websocketChannel{
-			{
-				Name:       channelToSubscribe.Channel,
-				ProductIDs: []string{channelToSubscribe.Currency.String()},
-			},
-		},
+	var unSub wsSub
+	unSub.Operation = "unsubscribe"
+	unSub.Arguments = []string{channelToSubscribe.Channel}
+	return b.WebsocketConn.SendMessage(unSub)
+}
+
+// Pinger pings
+func (b *BTSE) Pinger() {
+	b.Websocket.Wg.Add(1)
+
+	defer b.Websocket.Wg.Done()
+	count := 57 * time.Second
+	timer := time.NewTimer(count)
+
+	for {
+		select {
+		case <-b.Websocket.ShutdownC:
+			return
+
+		case <-timer.C:
+			b.WebsocketConn.Connection.WriteMessage(websocket.TextMessage, []byte("ping"))
```

Comment:
The BTSE docs say to send a ping frame message
https://tools.ietf.org/html/rfc6455#section-5.5.2 
This is a different type of control message used by websocket protocol
Gorilla supports it as well https://github.com/gorilla/websocket/blob/5b740c29263eb386f33f265561c8262522f19d37/conn.go#L77

It would be better off to use a [ticker](https://golang.org/pkg/time/#NewTicker) here instead of timer while not clearly stated timers are best used when you need to do something once after a duration tickers are if you need to do something repeatedly and wrap it in a check that the websocket is connected 

Also is the waitgroup is necessary here?